File: Leetcode-Problems/Find Unique Binary String.cpp
class Solution {
    public:
        string findDifferentBinaryString(vector<string>& nums) {
            string ans = "";
            for (int i = 0; i < nums.size(); i++) {
                int temp = nums[i][i];
                if (temp == '1') {
                    ans += '0';
                } else {
                    ans += '1';
                }
            }
            return ans;
        }
    };

File: Leetcode-Problems/Find Unique Binary String.java
class Solution {
    public String findDifferentBinaryString(String[] nums) {
        String ans = "";
        for (int i = 0; i < nums.length; i++) {
            char ch = nums[i].charAt(i);
            ans += ch == '1' ? '0' : '1';
        }
        return ans;
    }
}